tutor/specs/components/scores/__snapshots__/index.spec.cjsx.snap

```diff
@@ -67,18 +65,16 @@ exports[`Scores Report renders and matches snapshot 1`] = `
                 1st
               </span>
             </div>
-          </a>
+          </h2>
         </li>
         <li
           className=""
-          tabIndex={1}
+          role="tab"
+          tabIndex={0}
         >
-          <a
+          <h2
             aria-selected={undefined}
```

Comment:
Hm is it acceptable for accessibility to have a clickable `h2` element?  If it is should we add `aria-role=button` ?